### PR TITLE
Update website

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

Update website with content related to Podman Desktop v1.10

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

**Review by cdrage — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]
Tested all the pages, LGTM.

**Comment by benoitf** [comment]
build is failing with a broken link

**Comment by deboer-tim** [comment]
> build is failing with a broken link

That's the link to install AI Lab from the catalog (extension/redhat.ai-lab). Ideally we'd have something that would validate these? Not sure if there's a way to approve/force for now.

File: website/docs/ai-lab/installing.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description: Podman AI Lab can help you run large language models (LLMs) locally with no pain.
+keywords: [podman desktop, podman, ai, llm, generative ai]
+tags: [ai, llm, generative ai]
+---
+
+# Installing the Podman AI Lab extension
+
+#### Procedure
+
+- If you have Podman Desktop already installed, click [here](podman-desktop:extension/redhat.ai-lab) to launch the installation of Podman AI Lab in Podman Desktop.
```
Comment (cdrage) [inline website/docs/ai-lab/installing.md]: this is what docusaurus is complaining about. 

unsure if it'll work, but I believe if it's URL it's ignored by docusaurus, so perhaps `./podman-desktop:extension/redhat.ai-lab` would work instead.

**Review by cdrage — CHANGES_REQUESTED** [review]

**Comment by feloy** [comment]
> > build is failing with a broken link
> 
> That's the link to install AI Lab from the catalog (extension/redhat.ai-lab). Ideally we'd have something that would validate these? Not sure if there's a way to approve/force for now.

I'm proposing to rewrite the link so the user needs to copy/paste the link, so the build would pass

**Review by cdrage — CHANGES_REQUESTED** [review]
@feloy 

Use:

```
<a href="podman-desktop:extension/redhat.ai-lab">here</a>
```

for now, that works and it will pass.


https://github.com/containers/podman-desktop/assets/6422176/11ae8056-a022-462f-83c1-4d609fe0684c

Also, the get started button on the main page, does not work?

**Review by cdrage — APPROVED** [review]

**Pull request merged**